tests/parametric/test_otel_span_methods.py

```diff
@@ -479,7 +480,10 @@ def test_otel_span_reserved_attributes_overrides(self, test_agent, test_library)
 
     @missing_feature(context.library < "java@1.25.0", reason="Implemented in 1.25.0")
     @missing_feature(context.library == "nodejs", reason="Not implemented")
-    @missing_feature(context.library == "dotnet", reason="Not implemented")
+    @missing_feature(
+        context.library == "dotnet",
+        reason=".NET does not honor 'something-else' as a 'false' value that would set 'analytics.event' as that would differ from the OTLP implementation.",
+    )
```

Comment:
From the RFC and OTLP code it doesn't appear that `something-else` would be treated as `false`, which shouldn't set the `_dd1.sr.eausr`

.NET passes all of the others besides the `something-else` case as it appears that only true/false values should be accepted as legitimate values that cause `_dd1.sr.eausr` to be set.